src/Symfony/Component/Notifier/Bridge/Lox24/Tests/Lox24TransportTest.php

```diff
@@ -0,0 +1,323 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\Notifier\Bridge\Lox24\Lox24Options;
+use Symfony\Component\Notifier\Bridge\Lox24\Lox24Transport;
+use Symfony\Component\Notifier\Bridge\Lox24\Type;
+use Symfony\Component\Notifier\Bridge\Lox24\VoiceLanguage;
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Message\PushMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Test\TransportTestCase;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+class Lox24TransportTest extends TransportTestCase
+{
+    public static function createTransport(?HttpClientInterface $client = null): Lox24Transport
+    {
+        $transport = new Lox24Transport('id:token', 'sender', ['type' => 'voice'], $client ?? new MockHttpClient());
+
+        return $transport->setHost('host.test');
+    }
+
+    public static function toStringProvider(): iterable
+    {
+        yield ['lox24://host.test?from=sender&type=voice', self::createTransport()];
+    }
+
+    public static function supportedMessagesProvider(): iterable
+    {
+        yield [new SmsMessage('+33611223344', 'Hello World!')];
+    }
+
+    public static function unsupportedMessagesProvider(): iterable
+    {
+        yield [new ChatMessage('Hello World!')];
+        yield [new PushMessage('subject', 'content')];
+    }
+
+    private const REQUEST_HEADERS = [
+        'X-LOX24-AUTH-TOKEN' => 'testToken',
+        'Accept' => 'application/json',
+        'Content-Type' => 'application/json',
+        'User-Agent' => 'LOX24 Symfony Notifier',
+    ];
+
+    private const REQUEST_BODY = [
+        'sender_id' => 'testFrom2',
+        'phone' => '+1411111111',
+        'text' => 'test text',
+        'is_text_deleted' => false,
+        'delivery_at' => 0,
+        'service_code' => 'direct',
+    ];
+
+    private MockObject|HttpClientInterface $client;
+
+    protected function setUp(): void
+    {
+        $this->client = $this->createMock(HttpClientInterface::class);
+    }
+
```

Comment:
const and setUp should be moved to the top of the class please

Reply:
fixed